### PR TITLE
Cleans up unused culture, factions, homeworlds and religions

### PR DESCRIPTION
**Pull request by ghost**
## About the Pull Request
This pull requests removes a lot of defines of unused culture, factions, religion and homeworlds. It also adds SCP related factions and religions for use when the culture system will be changed
## Why It's Good For The Game
The define changes are good for the codebase since i don't think anyone is going to spawn as a Unathi, and removing space references is always good for immersion.
## Changelog

:cl:
fix: Removed space or ss13 universe references from some objects
add: SCP related factions and religion defines and descriptions for use later
del: Removed unused factions, homeworlds, religions and culture defines and descriptions in the code
/:cl:

### CODE REVIEW

File: code/game/gamemodes/wizard/wizard.dm [hunk 1]
```diff
@@ -1,7 +1,7 @@
 /datum/game_mode/wizard
 	name = "Wizard"
-	round_description = "There is a SPACE WIZARD onboard. You can't let the magician achieve their objectives!"
-	extended_round_description = "A powerful entity capable of manipulating the elements around him, most commonly referred to as a 'wizard', has made their way onboard. They have a wide variety of powers and spells available to them that makes your own simple moral self tremble with fear and excitement. Ultimately, their purpose is unknown. However, it is up to you and your crew to decide if their powers can be used for good or if their arrival foreshadows devastation."
+	round_description = "There is a SPACE WIZARD on site. You can't let the magician achieve their objectives!"
```
Comment (tichys) [inline code/game/gamemodes/wizard/wizard.dm:3]: didn't knwo there were space wizards on earth

File: code/modules/culture_descriptor/culture/cultures_hidden.dm [hunk 2]
```diff
@@ -3,20 +3,6 @@
...
-/decl/cultural_info/culture/hidden/shadow
-	name =             CULTURE_STARLIGHT
-	language =         LANGUAGE_CULT
-	additional_langs = list(LANGUAGE_CULT_GLOBAL)
-
-/decl/cultural_info/culture/hidden/cultist
-	name =   CULTURE_CULTIST
-	language = LANGUAGE_CULT
-
 /decl/cultural_info/culture/hidden/cultist/get_random_name()
```
Comment (tichys) [inline code/modules/culture_descriptor/culture/cultures_hidden.dm]: Are we not cleaning up additional procs that have no parents?

File: code/game/gamemodes/wizard/wizard.dm (hunk 1 above)
Comment (ghost) [reply code/game/gamemodes/wizard/wizard.dm:3]: i didnt know what to put tbh

File: code/modules/culture_descriptor/culture/cultures_hidden.dm (hunk 2 above)
Comment (ghost) [reply code/modules/culture_descriptor/culture/cultures_hidden.dm]: its fixed sirr

File: code/game/objects/items/weapons/storage/toolbox.dm [hunk 3]
```diff
@@ -45,7 +45,7 @@
 
 /obj/item/storage/toolbox/mechanical
 	name = "mechanical toolbox"
-	desc = "Bright blue toolboxes like these are one of the most common sights in maintenance corridors on virtually every ship in the galaxy."
+	desc = "A highly durable and sturdy toolbox designed to provide secure and organized storage for a wide range of tools."
```
Comment (Tsurupeta) [inline code/game/objects/items/weapons/storage/toolbox.dm]: delete this line, it's inherited from parent

File: code/game/objects/items/weapons/storage/toolbox.dm [hunk 4]
```diff
@@ -61,7 +61,7 @@
 
 /obj/item/storage/toolbox/electrical
 	name = "electrical toolbox"
-	desc = "Bright yellow toolboxes like these are one of the most common sights in maintenance corridors on virtually every ship in the galaxy."
+	desc = "A highly durable and sturdy toolbox designed to provide secure and organized storage for a wide range of tools."
```
Comment (Tsurupeta) [inline code/game/objects/items/weapons/storage/toolbox.dm]: this too

File: code/modules/codex/entries/misc.dm [hunk 5]
```diff
@@ -16,7 +16,7 @@
 	associated_paths = list(/obj/item/gun/launcher/money)
 	mechanics_text = "Load money into the cannon by picking it up with the gun, or feeding it directly by hand. Use in your hand to configure how much money you want to fire per shot."
 	lore_text = "These devices were invented several centuries ago and are a distinctly human cultural infection. \
-	They have produced knockoffs as timeless and as insipid as the potato gun and the paddle ball, showing up in all corners of the galaxy. \
+	They have produced knockoffs as timeless and as insipid as the potato gun and the paddle ball, showing up in all four corners of the Earth. \
```
Comment (Tsurupeta) [inline code/modules/codex/entries/misc.dm]: Spheres don't really have corners...

File: code/modules/culture_descriptor/culture/cultures_adherent.dm [hunk 6]
```diff
@@ -1,17 +1,3 @@
...
-	and disoriented for hundreds of years following. Now in contact with humanity, the Vigil is tentatively making inroads towards \
-	a place in the wider galactic culture."
-	hidden_from_codex = TRUE
-	language = LANGUAGE_ADHERENT
-	secondary_langs = list(
-		LANGUAGE_HUMAN_GERMAN,
-		LANGUAGE_ENGLISH
-	)
-
 /decl/cultural_info/culture/adherent/get_random_name(gender)
```
Comment (Tsurupeta) [inline code/modules/culture_descriptor/culture/cultures_adherent.dm]: delete unused procs

File: code/modules/culture_descriptor/culture/cultures_ascent.dm [hunk 7]
```diff
@@ -86,19 +86,6 @@ GLOBAL_LIST_INIT(worker_names, list(
...
-	additional_langs = list(LANGUAGE_MANTID_BROADCAST, LANGUAGE_MANTID_VOCAL, LANGUAGE_NABBER, LANGUAGE_SKRELLIAN)
-	hidden = TRUE
-	description = "The Ascent is an ancient, isolated stellar empire composed of the mantid-cephalopodean \
-	Kharmaani, the Monarch Serpentids, and their gaggle of AI servitors. Day to day existence in the Ascent is \
-	largely a matter of navigating a bewildering labyrinth of social obligations, gyne power dynamics, factional \
-	tithing, protection rackets, industry taxes and plain old interpersonal backstabbing. Both member cultures of \
-	this stellar power are eusocial to an extent, and their society is shaped around the teeming masses \
-	of workers, soldiers, technicians and 'lesser' citizens supporting a throng of imperious and all-powerful queens."
-
 /decl/cultural_info/culture/ascent/get_random_name(gender)
```
Comment (Tsurupeta) [inline code/modules/culture_descriptor/culture/cultures_ascent.dm]: this too

File: code/modules/culture_descriptor/culture/cultures_ipc.dm [hunk 8]
```diff
@@ -1,20 +1,2 @@
...
-		LANGUAGE_HUMAN_CHINESE,
-		LANGUAGE_HUMAN_ARABIC,
-		LANGUAGE_HUMAN_INDIAN,
-		LANGUAGE_HUMAN_SPANISH,
-		LANGUAGE_HUMAN_RUSSIAN,
-		LANGUAGE_ENGLISH,
-		LANGUAGE_SIGN
-	)
-
 /decl/cultural_info/culture/ipc/sanitize_name(new_name)
```
Comment (Tsurupeta) [inline code/modules/culture_descriptor/culture/cultures_ipc.dm]: this too

**Review by Tsurupeta — DISMISSED** [review]
Delete the files that you've left empty after removing all of their contents...

File: code/modules/clothing/under/accessories/badges.dm [hunk 9]
```diff
@@ -180,15 +180,15 @@
 	icon_state = "agentbadge"
 	slot_flags = SLOT_BELT | SLOT_TIE | SLOT_MASK
 	slot = ACCESSORY_SLOT_INSIGNIA
-	badge_string = FACTION_SPACECOPS
+	badge_string = "Spacecops"
```
Comment (TheDarkElites) [inline code/modules/clothing/under/accessories/badges.dm]: You can totally remove any badges that used removed factions.

File: code/modules/clothing/under/accessories/badges.dm [hunk 10]
```diff
@@ -180,15 +180,15 @@
...
 
 /obj/item/clothing/accessory/badge/tracker
```
Comment (TheDarkElites) [inline code/modules/clothing/under/accessories/badges.dm]: same here

**Review by TheDarkElites — CHANGES_REQUESTED** [review]

**Review by TheDarkElites — APPROVED** [review]
Looks good ill slot it for testmerge

**Comment by TheDarkElites** [comment]
tmed fine and nothing broke

**Pull request merged**